client/gatsby-ssr.js

```diff
@@ -32,3 +32,19 @@ export const onRenderBody = ({
   setHeadComponents(getheadTagComponents());
   setPostBodyComponents(getPostBodyComponents(pathname));
 };
+
+export const onPreRenderHTML = ({
+  getHeadComponents,
+  replaceHeadComponents
+}) => {
+  const headComponents = getHeadComponents();
+  headComponents.sort((x, y) => {
+    if (x.key === 'boostrap-min-preload' || x.key === 'boostrap-min') {
+      return -1;
+    } else if (y.key === 'TypographyStyle' || x.key === 'boostrap-min') {
```

Comment:
```suggestion
    } else if (y.key === 'TypographyStyle') {
```

I do not think this is ever reached

Reply:
```suggestion
    } else if (y.key === 'TypographyStyle' || y.key === 'boostrap-min') {
```
Actually, I am wondering if it is supposed to be the above?

Reply:
I added the TypographyStyle by mistake.